webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
-                      aria-label="Audits"
-                    >
-                      {audit &&
-                        Object.entries(audit.audited_changes).map(
-                          ([key, value], i) => (
-                            <DataListItem key={`audit-${i}`} aria-labelledby="">
-                              <DataListItemRow>
-                                <DataListItemCells
-                                  dataListCells={[
-                                    <DataListCell key={key}>
-                                      <span> {key}</span>
-                                    </DataListCell>,
-                                    <DataListCell
-                                      key={`old-value-${i}`}
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                    >
-                                      {value && (
-                                        <ElipsisWithTooltip>
-                                          <mark
-                                            style={{
-                                              backgroundColor: '#FFC0CB',
-                                            }}
-                                          >
-                                            {value[0]}
-                                          </mark>
-                                        </ElipsisWithTooltip>
-                                      )}
-                                    </DataListCell>,
-                                    <DataListCell
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                      key={`new-value-${i}`}
-                                    >
-                                      {value && (
-                                        <mark
-                                          style={{
-                                            backgroundColor: '#90EE90',
-                                          }}
-                                        >
-                                          {value[1]}
-                                        </mark>
-                                      )}
-                                    </DataListCell>,
-                                  ]}
-                                />
-                              </DataListItemRow>
-                            </DataListItem>
-                          )
-                        )}
-                    </DataList>
-                  </AccordionContent>
-                </AccordionItem>
-              );
-            return null;
-          })}
-        </Accordion>
+        <SkeletonLoader
+          skeletonProps={{ count: 3 }}
```

Comment:
if you decide to go with a constant for the number of audits fetched, it should probably be used here as well